Header header

float32[] point_range_2D
sensor_msgs/PointCloud2 origin_cloud
sensor_msgs/PointCloud2 extract_cloud
